### DISCUSSION TITLE
How to include schema.prisma in TypeScript output directory

### DISCUSSION THREAD

**Discussion by blevine** [prisma/prisma#8752] [Q&A] [answered]
I have a TypeScript project that uses Prisma.  The tsc command only copies .ts and .js files to the output directory.  What are folks doing to include the schema.prisma file in the output so it can be read at runtime?  Anything more elegant than just copying the schema.prisma file to the output directory via a script?

**Comment by ryands17** [comment]
@blevine 👋 

I can't think of any way apart from explicitly adding a copy statement to copy the `prisma folder`. I personally don't need to do this in a normal environment as I keep `prisma` in the root and not in `src` which makes it easier but I assume your setup might be different.

**Comment by blevine** [comment, marked as answer]
I actually found this [tsc-hooks](https://github.com/swimauger/tsc-hooks) project which is turning out to be pretty helpful.  It's a little buggy, but it works. Make sure to pull the 1.0.8 version as the author recommends in the README.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
